### PR TITLE
Connect DD 

### PR DESCRIPTION
**Pull request by gustafguner**
Prompts the user to connect their direct debit after having signed up.

⚠️ Need english translations for the text keys 
⚠️ Dependent on: https://github.com/HedvigInsurance/payment-service/pull/88.

<div style="width: 100%">

</div>

### CODE REVIEW

File: src/pages/ConnectPayment/containers/RegisterDirectDebitMutation.tsx [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+    registerDirectDebit(clientContext: $clientContext) {
+      url
+    }
+  }
+`
+
+interface RegisterDirectDebitData {
+  registerDirectDebit: {
+    url: string
+    orderId: string
```
[the remark this answers is not in the document]
Comment (gustafguner) [reply src/pages/ConnectPayment/containers/RegisterDirectDebitMutation.tsx]: Hm, in what query? 🤔

File: src/pages/ConnectPayment/components/TrustlyRetryPage.tsx [hunk 2]
```diff
@@ -0,0 +1,32 @@
+import { colors } from '@hedviginsurance/brand'
```
[the remark this answers is not in the document]
Comment (gustafguner) [reply src/pages/ConnectPayment/components/TrustlyRetryPage.tsx]: Yeah def 👍

File: .gitignore [hunk 3]
```diff
@@ -2,4 +2,5 @@
 node_modules
 yarn-error.log
 /coverage/*
+.env
```
[the remark this answers is not in the document]
Comment (gustafguner) [reply .gitignore]: Hehe no it wasn't 😅

File: src/pages/Download/sections/DownloadApp.tsx [hunk 4]
```diff
@@ -45,85 +55,307 @@ const Column = styled('div')({
...
+                      let phoneNumber = form.phoneNumber.trim()
+
+                      if (!phoneNumber) {
+                        return
+                      }
+
+                      setIsSending(true)
+
+                      if (!phoneNumber.match(/^\+/)) {
+                        phoneNumber = `+46${phoneNumber}`
```
[the remark this answers is not in the document]
Comment (gustafguner) [reply src/pages/Download/sections/DownloadApp.tsx]: Yeah that's true. It came straight from `web-next` so perhaps we should fix that there too.

File: src/pages/ConnectPayment/containers/RegisterDirectDebitMutation.tsx (hunk 1 above)
[the remark this answers is not in the document]
Comment (palmenhq) [reply src/pages/ConnectPayment/containers/RegisterDirectDebitMutation.tsx]: Right above :) L9-L11

File: src/pages/ConnectPayment/components/TrustlyModal.tsx [hunk 5]
```diff
@@ -0,0 +1,122 @@
...
+                  return
+                }
+
+                const contentWindow = iframe.contentWindow
+
+                if (!contentWindow) {
+                  return
+                }
+
+                const href = contentWindow.location.href
```
[the remark this answers is not in the document]
Comment (gustafguner) [reply src/pages/ConnectPayment/components/TrustlyModal.tsx]: Yeah changed it to something like that 👌

File: src/pages/ConnectPayment/components/TrustlyFailPage.tsx [hunk 6]
```diff
@@ -0,0 +1,47 @@
+import { colors } from '@hedviginsurance/brand'
+import { TranslationsConsumer } from '@hedviginsurance/textkeyfy'
+import * as React from 'react'
+import styled from 'react-emotion'
+import { Button } from '../../../components/buttons'
+import { CurrentLanguage } from '../../../components/utils/CurrentLanguage'
+
+const Wrapper = styled('div')({
+  width: '100%',
+  height: '100vh',
```
[the remark this answers is not in the document]
Comment (gustafguner) [reply src/pages/ConnectPayment/components/TrustlyFailPage.tsx]: 👍

File: src/pages/ConnectPayment/containers/RegisterDirectDebitMutation.tsx (hunk 1 above)
[the remark this answers is not in the document]
Comment (gustafguner) [reply src/pages/ConnectPayment/containers/RegisterDirectDebitMutation.tsx]: 🤦‍♂ I blame my cold!

**Comment by gustafguner** [comment]
@palmenhq Yes, let's test it a bit in staging before deploying to prod 😄

**Pull request merged**